### PR TITLE
Update NOTICE.txt

### PR DESCRIPTION
**Pull request by amyblais**
Based on https://github.com/mattermost/desktop/blob/master/package.json

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -734,48 +870,6 @@ provides access to the windows registry through the REG tool, with additional un
 
 * LICENSE: BSD-2-Clause
 
-
-
----
-
-## wix
```
Comment (devinbinnie) [inline NOTICE.txt:741]: We still do use WiX for the MSI build, so I don't think this should be removed?

File: NOTICE.txt [hunk 2]
```diff
@@ -452,160 +3505,139 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
...
+Find potentially unused exports in your Typescript project with zero configuration.
 
 * HOMEPAGE:
-  * https://github.com/atlassian/react-beautiful-dnd
+  * https://github.com/nadeesha/ts-prune
 
 * LICENSE: 
 
-Copyright 2019 Atlassian Pty Ltd
+---
```
Comment (amyblais) [inline NOTICE.txt:3523]: Does this have a license?

File: NOTICE.txt [hunk 3]
```diff
@@ -9,33 +9,2585 @@ This document includes a list of open source components used in Mattermost Deskt
...
+
+This product contains 'electron-connect' by Yosuke Kurami.
+
+Utility tool to develop applications with Electron.
+
+* HOMEPAGE:
+  * https://github.com/Quramy/electron-connect
+
+* LICENSE: 
+
```
Comment (amyblais) [inline NOTICE.txt:1533]: Does this have a license?

**Review by m1lt0n — APPROVED** [review]

File: NOTICE.txt (hunk 3 above)
Comment (devinbinnie) [reply NOTICE.txt:1533]: Yep, MIT. https://github.com/Quramy/electron-connect#license

**Review by devinbinnie — APPROVED** [review]

**Comment by amyblais** [comment]
@devinbinnie I don't have merge permissions - please help merge and cherry-pick to ``release-5.3``.

**Pull request merged**